modules/nw-rfhe-creating-hardware-event.adoc

```diff
@@ -78,6 +75,45 @@ If `storageType` is not set, an error occurs.
 $ oc create -f hardware-event.yaml
 ----
 
+. Create a `Service` resource for the events HTTP transport. For example:
```

Comment:
This step should be removed here. It is already documented in Step 3 of #cnf-migrating-from-amqp-to-http-transport_using-ptp